JuiceMaker/JuiceMaker/Controller/FruitStoreViewController.swift

```diff
@@ -16,11 +21,99 @@ final class FruitStoreViewController: UIViewController {
     
     @IBOutlet weak var strawberryStepper: UIStepper!
     @IBOutlet weak var bananaStepper: UIStepper!
-    @IBOutlet weak var pineapppleStepper: UIStepper!
+    @IBOutlet weak var pineappleStepper: UIStepper!
     @IBOutlet weak var kiwiStepper: UIStepper!
     @IBOutlet weak var mangoStepper: UIStepper!
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        setUpNavigationItem()
+        setUpLabel()
+        setUpStepper()
+    }
+    
+    func setUpNavigationItem() {
+        self.title = "재고 추가"
+        let rightButton = UIBarButtonItem(title: "닫기", style: .plain, target: self, action: #selector(goBackJuiceMakeView))
+        self.navigationItem.rightBarButtonItem = rightButton
+    }
```

Comment:
```suggestion
    func setUpNavigationItem() {
        self.title = "재고 추가"
        let rightButton = UIBarButtonItem(
            title: "닫기",
            style: .plain, 
            target: self, 
            action: #selector(goBackJuiceMakeView)
        )
        self.navigationItem.rightBarButtonItem = rightButton
    }
```

이런식으로 개행을 주어도 좋을 것 같아요 ☺️

Reply:
감사합니다!